docs/components/property-results.mdx

```diff
@@ -30,6 +30,31 @@ import { PropertyResults } from 'homeflowjs/properties';
 - `infiniteScroll` enables infinite scroll (automatically loading the next page of results when the user scrolls to the bottom of the results container). Default is `false`.
 - `noMap` - pass this if you do not want the `PropertyResults` component to include a map view (in case you want to render the map separately).
 
+#### The inserts prop
+
+:::caution
+
+The inserts prop is currently in alpha, therefor it does not support multiple components.
+
+:::
+
+- `inserts` is an array that allows you to pass a component to the `PropertyResults` component. This allows the 
+`PropertyResults` component to render components such as "banners" within the results grid. The array must contain 
```

Comment:
I would go for something like:

`> `inserts` can optionally be passed to `PropertyResults` as an array in order to render other components at certain intervals between the search results. This is commonly used to insert adverts, banners or call-to-actions.